### DISCUSSION TITLE
_regex permissions: help needed!

### DISCUSSION THREAD

**Discussion by carolin-skiply** [directus/directus#13315] [Archive]
Hi guys, 
I am trying to use the _regex validation in permissions settings but so far it's not working. 
I have a tags fiels that I want to match with the following regex: (BREST|CHOLET). 
Unfortunately, it's not filtering my list at the moment. 
Here is the permission settings from the api:
![image](https://user-images.githubusercontent.com/34483024/168562253-0675e1ef-20d9-4a66-b546-b587c49d3989.png)

Thanks for the help,
Caroline.

**Comment by rijkvanzanten** [comment]
Are you using `tags` as JSON or CSV?

**Reply by carolin-skiply** [reply]
I use it as CSV.

**Reply by carolin-skiply** [reply]
@rijkvanzanten any idea ? Tks.

**Comment by carolin-skiply** [comment]
Hi @rijkvanzanten, any idea what I am doing wrong? or maybe it is not possible to do this with the regex operator?
I am trying to calculate/set my permission on tags my user has. Is there any other way I can do that?
Thanks for the help,
Caroline.

**Reply by rijkvanzanten** [reply]
The `_regex` operation is only available in `validation`, not in `permissions` 👍🏻  https://docs.directus.io/configuration/filter-rules/#filter-operators

**Reply by carolin-skiply** [reply]
Thank you for getting back to me! I note that this is not available in `permissions`.
If it is not possible maybe the option should not be available
![image](https://user-images.githubusercontent.com/34483024/169965956-3b7aba16-b8f6-4459-9578-dc71561857cf.png)

**Reply by rijkvanzanten** [reply]
Tweaked in https://github.com/directus/directus/pull/13511 👍🏻
